eth-types/src/evm_types/opcode_ids.rs

```diff
@@ -346,6 +346,19 @@ impl OpcodeId {
                 | OpcodeId::STATICCALL
         )
     }
+
+    /// Returns `true` if the `OpcodeId` is a `CALL` or `CREATE` related .
+    pub fn is_call_or_create(&self) -> bool {
+        [
+            Self::CALL,
+            Self::DELEGATECALL,
+            Self::CALLCODE,
+            Self::STATICCALL,
+            Self::CREATE,
+            Self::CREATE2,
+        ]
+        .contains(self)
+    }
```

Comment:
maybe later we need a method is_call and is_create. So another approach here is to add `is_call` AND `is_create`. And use `is_call() || is_create()` in caller side.

Reply:
agree !

Reply:
updated in [2ade20d](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/647/commits/2ade20d8891a442aa89897ab594f2cd6779815d2) thanks !